public/content/it/welcome/install-d-locally.md

```diff
@@ -0,0 +1,39 @@
+# Come installare D sul proprio computer
+
+All'interno del sito internet del linguaggio D [dlang.org](https://dlang.org) è possibile [scaricare](http://dlang.org/download.html) l'ultima versione del compilatore di riferimento **DMD** (Digital Mars D):
+
+### Windows
+
+* Tramite il [programma di installazione](http://downloads.dlang.org/releases/2.x/2.071.0/dmd-2.071.0.exe)
+* oppure come [archivio compresso](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.windows.7z)
+* o tramite [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
+
+* Come [pacchetto `.dmg`](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.dmg)
+* oppure come [archivio compresso](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.osx.tar.xz)
+* o tramite [Homebrew](http://brew.sh): `brew install dmd`
```

Comment:
I just realized that we are hard-coding the latest release here, that will be fixed very soon (#449).
So how about applying `s/2.071.0/{{latest-release}}/g` with your favorite regex tool (e.g. `sed s/2.071.0/{{latest-release}}/g  -i <file> `)?